src/lib/utils/createSendEventForHits.ts

```diff
@@ -184,6 +190,7 @@ export function createBindEventForHits({
       index,
       methodName: 'bindEvent',
       args,
+      isSearchStalled: false,
```

Comment:
this is false, because if the search is stalled, you still want to bind events (for later)